src/qt/optionsmodel.cpp

```diff
@@ -372,6 +377,8 @@ QVariant OptionsModel::data(const QModelIndex & index, int role) const
             return settings.value("fListen");
         case Server:
             return settings.value("server");
+        case EnablePSBTControls:
```

Comment:
In commit "qt: hide Create Unsigned button behind an expert mode option" (790733e54c3daff547acbc90fb179b48b6d2fb3c)

Suggest moving this case up near `case CoinControlFeatures` above, so this is grouped together with GUI options, not node options and it is easier ensure that all these options are handled consistently in the future. Same suggestion as well for `OptionsModel::setData`

Reply:
Done